package.nls.json

```diff
@@ -263,6 +263,7 @@
     "cmake-tools.configuration.cmake.launchBehavior.description": "Controls what happens with the launch terminal when you launch a target.",
     "cmake-tools.configuration.cmake.automaticReconfigure.description": "Automatically configure CMake project directories when the kit or the configuration preset is changed.",
     "cmake-tools.configuration.cmake.pinnedCommands.description":"List of CMake commands to pin.",
+    "cmake-tools.configuration.cmake.enableKitScan.description": "Automatically scans for kits.",
```

Comment:
It might be helpful to be slightly more verbose here. 

See `cmake-tools.configuration.cmake.allowUnsupportedPresetsVersions.description`, `cmake-tools.configuration.cmake.enableTraceLogging.description`, and other enablement descriptions for inspiration. Maybe something like 

"Enable automatically scanning for kits on start-up" or something similar.